pkg/agents/team7/agent1/agent.go

```diff
@@ -144,7 +145,16 @@ func (a *CustomAgent7) Run() {
 
 		//has the agent seen the platform
 		if a.CurrPlatFood() != -1 && !a.seenPlatform {
-			a.foodEaten = a.TakeFood(foodtotake)
+			foodEaten, err := a.TakeFood(foodtotake)
```

Comment:
should be `a.foodEaten` instead of `foodEaten`

Reply:
Doesn't work unfortunately, I tried that before. I do set it later on